source/common/http/access_log/request_info_impl.h

```diff
@@ -54,6 +61,7 @@ struct RequestInfoImpl : public RequestInfo {
   uint64_t response_flags_{};
   Upstream::HostDescriptionConstSharedPtr upstream_host_{};
   bool hc_request_{};
+  std::string downstream_address_;
```

Comment:
@mattklein123 the downstream_address_ is required on the request_info object because it gets accessed in the req_header_formatter.cc (line 79 [here](https://github.com/khgandhi/envoy/blob/43ccc9accf4aa6fd663914d84c5a3846d775bfe5/source/common/router/req_header_formatter.cc#L79)). I am not sure how would the formatter access the downstream_address_ if is set on the connection_manager.